### DISCUSSION TITLE
cleanup policy for MEND_RNV_LOG_HISTORY_S3 ?

### DISCUSSION THREAD

**Discussion by tomkuipers** [mend/renovate-ce-ee#492] [Q&A] [answered]
`MEND_RNV_LOG_HISTORY_CLEANUP_CRON` sets the cleanup interval and uses settings for   `MEND_RNV_LOG_HISTORY_TTL_DAYS` to clean log directory defined in `MEND_RNV_LOG_HISTORY_DIR`. Does this also apply for logs in `MEND_RNV_LOG_HISTORY_S3` ?

**Comment by nabeelsaabna** [comment, marked as answer]
Hi @tomkuipers 
when using `MEND_RNV_LOG_HISTORY_S3`, after successfully uploading the file to `s3` the local copy is deleted.
In other words, you can only use one of the two options :
- `MEND_RNV_LOG_HISTORY_DIR` 
- `MEND_RNV_LOG_HISTORY_S3` (no cleanup by the Mend Renovate CE/EE)

I see this is missing from the docs.
cc @justo-mend
